src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const discountSmall = 20;
+  const discountBig = 50;
+  let costRent = 40 * days;
+
+  if (days < 7 && days >= 3) {
+    costRent -= discountSmall;
+  } else if (days >= 7) {
+    costRent -= discountBig;
+  }
```

Comment:
```suggestion
  if (days < 7 && days >= 3) {
    return costRent - discountSmall;
  }
  
  if (days >= 7) {
    return costRent - discountBig;
  }
```
You can just immediately return the full price to make your code a bit cleaner 
![image](https://user-images.githubusercontent.com/96409563/225567882-d480a72e-1ec7-489e-849f-61de7c6849a8.png)